tests/e2e-leg-11/client-proxy/30-add-2-nodes-to-pri1.yaml

```diff
@@ -0,0 +1,25 @@
+# (c) Copyright [2021-2024] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertica.com/v1
+kind: VerticaDB
+metadata:
+  name: v-client-proxy
+spec:
+  subclusters:
+    - name: pri1
+      size: 5
+      type: primary
+      proxy:
+        image: kustomize-vproxy-image
+        replica: 2
```

Comment:
You are only changing the size, do you have to include the proxy? Also why is the proxy image kustomize-vproxy-image?  
Isn't this going to recreate a deployment with a wrong image.